### ISSUE TITLE
Inheriting from non-class should raise an error

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#331]
Originally reported by: **Antony Lee (BitBucket: [anntzer](http://bitbucket.org/anntzer), GitHub: @anntzer?)**

---

```
#!python
foo = 1

class C(foo): pass
```

Pylint could raise an error here.

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/331
